src/main/java/org/opensearch/security/authtoken/jwt/JwtVendor.java

```diff
@@ -142,7 +145,10 @@ public String createJwt(
             throw new Exception("Roles cannot be null");
         }
 
-        /* TODO: If the backendRoles is not null and the BWC Mode is on, put them into the "dbr" claim */
+        if (bwcModeEnabled && backendRoles != null) {
+            String listOfBackendRoles = String.join(",", backendRoles);
+            jwtClaims.setProperty("br", EncryptionDecryptionUtil.encrypt(claimsEncryptionKey, listOfBackendRoles));
```

Comment:
If this setting is enabled then send the backend roles unencrypted. This setting will determine whether backend roles (`br`) is included or excluded as a claim in the token.

Reply:
+1. Based on this TODO comment:
```
/* TODO: If the backendRoles is not null and the BWC Mode is on, put them into the "dbr" claim */
```
@samuelcostae Please update it to reflect non-encrypted backend role `dbr`